docs/figure_file_format.rst

```diff
@@ -0,0 +1,223 @@
+
+OMERO.figure files
+==================
+
+OMERO.figure files are JSON data, stored in OMERO as OriginalFiles linked to FileAnnotations.
+
+File Annotations
+----------------
+
+The FileAnnotations are identified as OMERO.figure files using a namespace of ```omero.web.figure.json```.
+
+The description of the FileAnnotation can be used to specify an Image to show as a thumbnail
+in the File > Open listing of files::
+
+    {
+        "imageId": 123,
+        // only needed if image comes from a different server
+        "baseUrl": "demo.openmicroscopy.org",
+    }
+
+In general, OMERO.figure FileAnnotations are *not* linked to any Images because this would
```

Comment:
This is probably also something we should review with a larger refactoring.

Reply:
Review how to prevent users delete the FileAnnotations?
There was a big discussion on this when I opened a webclient PR to allow this to be configured. PR got closed.

Reply:
PRs can always get re-opened. ;) But it would be part of a larger epic.